Outfit Selection/App/AppDelegate/AppDelegate+AppsFlyerLibDelegate.swift

```diff
@@ -11,10 +11,10 @@ import UIKit
 
 extension AppDelegate: AppsFlyerLibDelegate {
     /// AppsFlyer Dev Key found in AppsFlyer.com dashboard > App Settings > DevKey
-    static let appsFlyerDevKey = "..." // Replace with your AppsFlyer Dev Key
+    static let appsFlyerDevKey = "JX8UnQW4Zx2PxoLETzdBVY" // Replace with your AppsFlyer Dev Key
     
     /// AppsFlyer App ID found in AppsFlyer.com Dashboard > id in top left corner (without "id")
-    static let appleAppID = "..." // Replace with your App ID
+    static let appleAppID = "1479321113" // Replace with your App ID
```

Comment:
Need to remove from commit — replace with "...", commit, replace back with real keys, check that AppDelegate+AppsFlyerLibDelegate.swift is in .gitignore and run git update-index --assume-unchanged Outfit\ Selection/App/AppDelegate/AppDelegate+AppsFlyerLibDelegate.swift